grafana-plugin/src/plugin.json

```
@@ -186,6 +186,465 @@
       ]
     }
   ],
+  "roles": [
+    {
+      "role": {
+        "name": "plugins.app:grafana-oncall-app:admin-user-settings-admin",
+        "displayName": "",
+        "description": "",
+        "group": "OnCall",
```

As we discussed, the display name field is now the name field (the RBAC role name will be computed, see https://github.com/grafana/grafana/pull/58166), the group is automatically set to the plugin Name (here Grafana OnCall) and is not read anymore. :slightly_smiling_face: 
```suggestion
        "name": "Admin user settings admin"",
        "description": "",
```